lib/mix/tasks/kitto.gen.dashboard.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Mix.Tasks.Kitto.Gen.Dashboard do
+  use Mix.Task
+  import Mix.Generator
+
+  @shortdoc "Generates a new empty dashboard template"
+
+  @template Path.join Path.expand("./templates", __DIR__), "dashboard.html.eex"
+
+  @moduledoc """
+  Generates a new empty dashboard template
+
+  Usage:
+
+      $ mix kitto.gen.dashboard all_the_data
+      # generates `dashboards/all_the_data.html.eex`
+  """
+
+  def run(argv) do
+    case List.first(argv) do
+      nil ->
+        IO.puts """
+        No dashboard name provided.
+        Usage:
+
+            mix kitto.gen.dashboard sample
+        """
+        exit :no_dashboard
+      dashboard ->
```

Comment:
Can't switch the cases around, otherwise the first case will match everything, including nil.